src/atoms/Button/buttons.module.scss

```diff
@@ -98,11 +98,16 @@ $brand-white: color('primary-2');
 }
 
 .disabled {
+  color: color('neutral-2');
```

Comment:
Under [the new design for disabled buttons](https://www.figma.com/file/iVP8PicuOCAkB7NpswyAWK/Mortar---RCL-color-updates?node-id=39%3A966&viewport=373%2C65%2C0.5921875238418579), the font/icon color should be darker.
![image](https://user-images.githubusercontent.com/6582067/106980772-ca974580-672e-11eb-85b1-879df55516cf.png)

Reply:
is the lock part of the new branding?